src/util/file-metadata.ts

```diff
@@ -1,12 +1,12 @@
-import { FileInfo } from './file.types';
-import { getIconForFile } from './icons';
-import { getIconFillColorForFile } from './icon-fill-colors';
-import { getIconBackgroundColorForFile } from './icon-background-colors';
+import { FileInfo } from 'src/components/file/file.types';
```

Comment:
Huh… is this path relative to the package root? I've never seen that before… 😮

Reply:
I hadn't pushed the latest, sorry 🙈 
The latest should be updated with correct imports.